comein/README.md

```diff
@@ -0,0 +1,6 @@
+# ComeIn NRW 
+
+The w3id.org URIs are used as permanent URIs for metadata specifications
+
+* Project: [ComeIn NRW](https://www.uni-due.de/comein)
+* Contact: [Manuel Oellers](https://www.uni-due.de/comein/metaportal/)
```

Comment:
If possible could you add the github maintainer id here too?  It was in the PR commit message, but we'd check here first.